django/utils/decorators.py

```diff
@@ -17,13 +17,27 @@ def __get__(self, instance, owner):
         return super(classonlymethod, self).__get__(instance, owner)
 
 
-def method_decorator(decorator):
+def method_decorator(decorator, name=""):
     """
     Converts a function decorator into a method decorator
     """
     # 'func' is a function at the time it is passed to _dec, but will eventually
     # be a method of the class it is defined on.
```

Comment:
Ah, sorry, I had completely missed your point. I have amended the comment in 5bcd875. Thanks for pointing it out again.